packages/node-runtime-worker-thread/src/worker-runtime.spec.ts

```diff
@@ -50,7 +49,14 @@ describe.skip('worker', () => {
 
   afterEach(async() => {
     if (worker) {
-      await worker.terminate();
+      // Possibly a Node.js bug that causes worker process to still be ref-ed
+      // after termination. To work around that, we are unrefing worker manually
+      // *immediately() after terminate method is called even though it should
```

Comment:
```suggestion
      // *immediately()* after terminate method is called even though it should
```

(I guess?)

Also, if you want to, feel free to add a reference to [https://github.com/nodejs/node/pull/37319](https://tinyurl.com/1bgd7wlb) here :)

Reply:
Oops, yeah, thanks! And totally will add a link. Wow, you fixed it in no time at all!

Reply:
I mean, you already did the hard part here :)